packages/gamut-labs/src/landingPage/PageFeatures.tsx

```diff
@@ -16,22 +16,25 @@ export type PageFeaturesProps = BaseProps & {
   /**
    * Array of features, which consist of image, image alt, title, and description
    */
-  features: Pick<
-    FeatureProps,
-    'imgSrc' | 'imgAlt' | 'title' | 'desc' | 'testId'
-  >[];
+  features: Omit<FeatureProps, 'featuresMedia' | 'onAnchorClick'>[];
 
   /**
-   * Whether an icon or a full size image should be rendered
+   * DEPRECATED: Whether an icon or a full size image should be rendered
```

Comment:
This way, the text will have a strikethrough in VS Code 😎 

```suggestion
   * @deprecated - Whether an icon or a full size image should be rendered
```